microsoft/testsuites/ltp/ltp.py

```diff
@@ -125,7 +125,7 @@ def run_test(
         self.run(parameters, sudo=True, force_run=True, timeout=12000)
 
         # to avoid no permission issue when copying back files
-        self.node.tools[Chmod].update_folder("/opt/ltp", "a+rwX", sudo=True)
+        self.node.tools[Chmod].update_folder("/home/lisatest", "a+rwX", sudo=True)
```

Comment:
change "a+rwX" => "a+rwx" should be enough, can you please update this PR? @r-dailey

Reply:
That did not resolve the permissions issue.

Reply:
@r-dailey this one should work
```self.node.tools[Chmod].update_folder("/opt", "a+rwx", sudo=True)```

Reply:
That is what I tried initially but @squirrelsc requested I do not use that form because that is modifying the permissions of a system folder and could influence other tests.

Reply:
> denied issue when use the test user (non-root) to copy back file from vm into local machine

Can we use test case working folder instead of /opt?

Reply:
That is what the current changes in this PR are doing. I moved the log output location to the working directory, and I changed the permissions command to allow copyback to use that location as well. If the permissions should not need to be changed for that directory, then there may be a separate issue that needs to be addressed.

Reply:
You can change it to test case working folder, instead of the home root folder. I think there shouldn't have permissions issue. There are some test cases to copy files back from there.

Reply:
Can you provide an example of the working folder?

Reply:
/opt folder is generated by compiling ltp. even we switch LTP_RESULT_PATH LTP_OUTPUT_PATH  to use user home folder, we still need change the permission, because mariner image default umask value is 027 and ltp run as sudo, most of linux distro umask value is 022. Now the most convenient way is to change permission for opt.

Reply:
Can it just change the subfolder in /opt, not change the whole folder?

Reply:
that is what current code does, it doesn't work.

Reply:
if so, we can give /opt more permission.

Reply:
Changed to just modifying the permissions of /opt